packages/TorneloScoresheet/src/components/OptionPopup/OptionPopup.tsx

```diff
@@ -0,0 +1,56 @@
+import { View } from 'react-native';
+import { SvgProps } from 'react-native-svg';
+import Sheet from '../Sheet/Sheet';
+import { styles } from './style';
+import { colours } from '../../style/colour';
+import PrimaryText, { FontWeight } from '../PrimaryText/PrimaryText';
+import React from 'react';
+import TextIconButton from '../TextIconButton/TextIconButton';
+
+type Option = {
+  icon?: React.FC<SvgProps>;
+  text?: string;
+  onPress: () => void;
+};
+type OptionPopupParams = {
+  message: string;
+  options: Option[];
+  visible: boolean;
+  onCancel: () => void;
+};
+
+const OptionPopup: React.FC<OptionPopupParams> = ({
```

Comment:
To me it makes more sense to name this `OptionSheet`, since that is the terminology we have used for the `Sheet` component. (otherwise we can rename `Sheet` to `Popup` - either works)

Reply:
Makes sense i'd just never heard the term sheet (I normally use the term 'modal' i think that comes from boostrap) Lets stick with sheet throughout the project

Reply:
FYI, modal is used as terminology in the main Tornelo system. Of course you are free to choose whatever you want.